bundle/manifests/submariner.clusterserviceversion.yaml

```diff
@@ -103,61 +121,71 @@ spec:
     For complete information about subctl, please refer to [this page](https://submariner.io/operations/deployment/subctl).
     In addition to Operator and subctl, Submariner also provides [Helm Charts](https://submariner.io/operations/deployment/helm).
 
```

Comment:
```suggestion
### Deployment
    Submariner provides an [Operator](https://github.com/submariner-io/submariner-operator) for easy API-based
    installation and management.
    A command line utility, [subctl](https://github.com/submariner-io/submariner-operator/releases), wraps the
    Operator to aid users with manual deployments and easy experimentation.
    subctl greatly simplifies the deployment of Submariner, and is therefore the recommended deployment method.
    For complete information about subctl, please refer to [this page](https://submariner.io/operations/deployment/subctl).
    In addition to Operator and subctl, Submariner also provides [Helm Charts](https://submariner.io/operations/deployment/helm).
```
@mangelajo `subctl` is mentioned here. There is a reference with a link for all sort of deployment we provide: subctl, helm,...
I think this specific description should focus on the bundle only and if the use wants to use other method he should follow our documentation.
Does it make sense?

Reply:
I want to highlight that they should use subctl as first option, unless they have a good reason to use the operator standalone. As it will save many general issues.

Do you think there is a better way to highlight it?

Reply:
In our current description we recommend for subctl:

> subctl greatly simplifies the deployment of Submariner, and is therefore the recommended deployment method

We can maybe rephrase the sentence to make the message even more clear.

Reply:
@mangelajo